VirtualReality/Widgets/DesignerPlugins/qMRMLSequenceBrowserPlayWidgetPlugin.cxx

```diff
@@ -0,0 +1,66 @@
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Copyright (c) Brigham and Women's Hospital
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+==============================================================================*/
+
+#include "qMRMLSequenceBrowserPlayWidgetPlugin.h"
+#include "qMRMLSequenceBrowserPlayWidget.h"
+
+//------------------------------------------------------------------------------
```

Comment:
@adamrankin  Here is the file @cpinter was referencing in his comment

Reply:
Yes I meant anything containing the word "Sequence" that is included in this PR.

Reply:
I... I didn't see this. I have no idea how this got added in.